hassio/src/addon-view/info/hassio-addon-info.ts

```diff
@@ -771,12 +781,51 @@ class HassioAddonInfo extends LitElement {
       };
       fireEvent(this, "hass-api-called", eventdata);
     } catch (err) {
-      this._error = `Failed to install addon, ${err.body?.message || err}`;
+      showAlertDialog(this, {
+        title: "Failed to install addon",
+        text: extractApiErrorMessage(err),
+      });
     }
-    this._installing = false;
+    button.progress = false;
   }
 
-  private async _uninstallClicked(): Promise<void> {
+  private async _startClicked(ev: CustomEvent): Promise<void> {
+    const button = ev.target as any;
+    button.progress = true;
+    try {
+      await validateHassioAddonOption(this.hass, this.addon.slug);
```

Comment:
This will return an error when not valid? That's a bit weird for a validation endpoint? I would expect a 200 response with a result that is either valid or not. What if the request errors for another reason like a bug?

Reply:
True!
I'll make a PR to adjust the backend